modules/print-styles.js

```diff
@@ -0,0 +1,70 @@
+/* @flow */
+
+var allPrintStyles = {};
+var listeners = [];
+
+var addListener = function (listener: Function): void {
+    var listenerIndex = listeners.indexOf(listener);
+
+    if (listenerIndex === -1) {
+        listeners.push(listener);
+    }
+};
+
+var removeListener = function (listener: Function): void {
+    var listenerIndex = listeners.indexOf(listener);
+
+    if (listenerIndex > -1) {
+        listeners.splice(listenerIndex, 1);
+    }
+};
+
+var emitChange = function () {
+    listeners.forEach(listener => listener());
+};
+
+var importantValues = function (styleObj) {
```

Comment:
rename: _appendImportant, or _appentImportantToEachValue

Also put `_` at the beginning of every non-exported function